Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7
+
+# Set up working directory
+WORKDIR /app
+
+# Copy everything into working directory
+COPY . /app
+
+# Install everything
+RUN apt-get update && apt-get install -y rubygems
+RUN pip install pipenv
+RUN pipenv install --dev
```

Comment:
This can just be `pipenv install` - the container is primarily for deployment, so we can probably drop the development dependencies.